src/components/fields/select-field.vue

```diff
@@ -4,34 +4,44 @@
       <FieldLabel :label="label" />
     </template>
 
-    <Select.Select v-model="selectedKey" :disabled="disabled">
-      <Select.SelectTrigger class="w-full">
-        <Select.SelectValue :placeholder="placeholder">
-          {{ selectedValue ? getLabelFromValue(selectedValue) : placeholder }}
-        </Select.SelectValue>
-      </Select.SelectTrigger>
-      <Select.SelectContent>
-        <slot name="select-top-content" />
+    <div>
+      <Select.Select
+        v-model="selectedKey"
+        :disabled="disabled"
+        @update:open="isDropdownOpen = $event"
+      >
+        <Select.SelectTrigger class="w-full">
+          <Select.SelectValue :placeholder="placeholder">
+            {{ selectedValue ? getLabelFromValue(selectedValue) : placeholder }}
+          </Select.SelectValue>
+        </Select.SelectTrigger>
+        <Select.SelectContent>
+          <div class="p-2">
+            <input-field v-model="searchQuery" type="text" placeholder="Search..." @keydown.stop />
```

Comment:
Why `@keydown.stop` needed?
Field should also be visible by condition. It should be available everywhere

Reply:
@keydown.stop is needed to avoid blocking input events in Select.Content.

Reply:
This doesn't explain anything to me. Share a more descriptive explanation or links to docs

Reply:
@keydown.stop is used to be able to enter the characters themselves into the input, because most likely it is connected with the focus control events in Radix (from which this Select comes, well, like schad-cn based on it)

In SelectContent, as I understand it, there is, what can I call it, a filter when there is focus, that is, when you are in focus on the content, you press some letter, and it throws you to the first element with this letter